Content.Server/Kitchen/EntitySystems/KitchenSpikeSystem.cs

```diff
@@ -127,10 +131,14 @@ private void OnInteractUsing(EntityUid uid, KitchenSpikeComponent component, Int
             UpdateAppearance(uid, null, component);
 
             _popupSystem.PopupEntity(Loc.GetString("comp-kitchen-spike-kill", ("user", Identity.Entity(userUid, EntityManager)), ("victim", victimUid)), uid, PopupType.LargeCaution);
-
+            
+            _transform.SetCoordinates(victimUid, Transform(uid).Coordinates);
             // THE WHAT?
             // TODO: Need to be able to leave them on the spike to do DoT, see ss13.
-            EntityManager.QueueDeleteEntity(victimUid);
+            var gibs = _bodySystem.GibBody(victimUid);
+            foreach (var gib in gibs) {
+                QueueDel(gib);
+            }
```

Comment:
This looks dodgy?

Reply:
Not sure what else can I do. Make a new method in body system that gibs body and deletes gibs afterwards?

Reply:
Why are the gibs even being spawned if we're just trashing them anyway.

Reply:
Because that's how GibBody works, gibs won't even spawn or you won't notice it.

Reply:
Spawning an entity and then immediately deleting it is shitcode and in this case isn't necessary.

Reply:
ok, actually all organs are created on body init, so GibBody doesn't spawn new entities. So that is appropriate.

Reply:
Yeah I had a look in my IDE and it looks okay though queuedel shouldn't be necessary as none of them are the root entity from what I could see.

Reply:
Yes, but when you put entity on spike full body will remain there (though we have only monkey sprite). And the question is where all gibs (e.g. chest) come from? 
Imo, kitchen spike needs serious refactor and this is just a bandaid to fix it broken state as of right now.